lib/crawler/keyword/keyword_extractor.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Crawler.Keyword.KeywordExtractor do
+  @selectors %{
+    ad_top: "#tads",
+    ad_top_count: "div.pla-unit-container",
+    urls_ad_top: "a[data-impdclcc]",
+    non_ad_count: "#res div.v7W49e",
+    urls_non_ad: ".yuRUbf > a",
+    total: "a[href]"
+  }
+
+  def parse(html) do
+    {_, document} = Floki.parse_document(html)
```

Comment:
should we check the status of parsing instead of ignoring it with `_`? and correctly handle parsing error

Reply:
Fixed in [07a8741](https://github.com/vnntsu/elixir-ic/pull/42/commits/07a87416987d0efc8a481d467ded3b5c59057dbc)